packages/flame/lib/src/game/game.dart

```diff
@@ -35,6 +35,16 @@ abstract mixin class Game {
   late final GestureDetectorBuilder gestureDetectors =
       GestureDetectorBuilder(refreshWidget)..initializeGestures(this);
 
+  /// Whether the app is currently in the foreground.
+  ///
+  /// This is used to discard the first [update] call after the app was in the
+  /// background to avoid a massive `dt` value.
+  ///
+  /// If you make use of this value,
+  /// remember to set this back to `true` when the app is resumed.
+  /// If you are using `Forge2DGame`, this is handled automatically.
+  bool isAppInForeground = true;
```

Comment:
So far we've only discussed making this change for the flame_forge2d plugin so I've only implemented it in the `Forge2DGame` class.
Would you like me to implement it in any `FlameGame` class?

Reply:
Yeah, sounds like a good idea, it could be done in `FlameGame`, but the default should be that it doesn't pause the game loop. :) Did it work in the browser?